include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -1028,9 +1028,9 @@ struct OrtApi {
   ORT_API2_STATUS(SetLanguageProjection, _In_ const OrtEnv* ort_env, _In_ OrtLanguageProjection projection);
 
   /**
-   * \param out is set to the nanoseconds of profiling's start time
+   * \param out is set to the microseconds of profiling's start time
    */
-  ORT_API2_STATUS(SessionGetProfilingStartTimeNs, _In_ const OrtSession* sess, _Outptr_ uint64_t* out);
+  ORT_API2_STATUS(SessionGetProfilingStartTimeMicros, _In_ const OrtSession* sess, _Outptr_ uint64_t* out);
```

Comment:
This is an ABI incompatible change. We cannot change the names of functions since this was already published as part of 1.5.1 rel.

Reply:
Thank you for catching this. It has been changed back to Ns with comments and this PR is ready for review.